composer.json

```
@@ -50,7 +50,7 @@
         "contao-components/swipe": "^2.0.3",
         "contao-components/tablesort": "^4.0",
         "contao-components/tablesorter": "^2.1",
-        "contao-components/tinymce4": "^4.7 || ^5.0",
+        "contao-components/tinymce4": "^4.7 || ^5.0 || ^6.0",
```

```suggestion
        "contao-components/tinymce4": "^5.0 || ^6.0",
```
Should we drop tinyMCE 4 support in Contao 5.0?

---

Yes, good idea. Changed in 7d18822c062bab1b499a7cce26f219b24cf5d4c6.